src/zoid/buttons/util.js

```diff
@@ -74,3 +74,35 @@ export function getVenmoExperiment(experiment : ?Experiment) : VenmoExperiment {
         enableVenmo: Boolean(isExperimentEnabled || isEnableFundingVenmo)
     };
 }
+
+export function applePaySession() : Function {
+    try {
+        if (!window.ApplePaySession) {
+            return undefined;
```

Comment:
I think we can just use `return;` and that will return undefined.

```suggestion
            return;
```